File: 2023/php/test/TestDay5B.php
<?php declare(strict_types=1);

$loader = require __DIR__ . '/../vendor/autoload.php';

use day5\b\Solution5B;
use PHPUnit\Framework\TestCase;
use utils\TestLogger;

final class TestDay5B extends TestCase
{
    public function testExample(): void
    {
        $this->assertSame(46, Solution5B::getResult("../resources/5/test_b.txt", TestLogger::createTestLogger()));
    }

    public function testResult(): void
    {
        $this->assertSame(125742456, Solution5B::getResult("../resources/5/input.txt", TestLogger::createTestLogger()));
    }

}

File: 2023/java/src/test/java/org/frieder/aoc/day5/TestDay5B.java
package org.frieder.aoc.day5;

import org.frieder.aoc.day5.b.Solution5B;
import org.junit.Test;

import java.io.IOException;

import static org.junit.Assert.assertEquals;

public class TestDay5B {
    @Test
    public void testExample() throws IOException {
        assertEquals(46D, Solution5B.getResult("../resources/5/test_b.txt"),0);
    }

    @Test
    public void testResult() throws IOException {
        assertEquals(125742456D, Solution5B.getResult("../resources/5/input.txt"),0);
    }
}
